android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/MainActivity.java

```diff
@@ -984,6 +1000,7 @@ public void run() {
                 }
 
                 //First get the new list into an ArrayMap so we can index on the cloudletName
+                //In this loop, we will also reset any existing cloudlet marker's icon, title, and snippet.

```

Comment:
I get the general flow, but maybe a util function to "clear" if useful in other contexts instead of inline here.